### DISCUSSION TITLE
Loading Screen?

### DISCUSSION THREAD

**Discussion by mathieugoodman** [vicwomg/pikaraoke#204] [Ideas]
Hello!

I was wondering if you have any suggestions on how to get a loading screen to show up from the time the RPi boots to the time the PiKaraoke splash screen shows up. There's a significant delay in my case due to the large library of MP3+G songs, and I can see friends and family getting confused by the RPi logo and CLI logs.

I tried some online tutorials and was able to hide the logs, but I've yet to get a loading screen to show up, even if it's a static picture.

Thanks!

**Comment by vicwomg** [comment]
Haven't confirmed this works, but have you tried this? 
https://techoverflow.net/2021/10/19/how-to-hide-all-boot-text-blinking-cursor-on-raspberry-pi/

Another option is to set the logo to a black square (or any other image). 
https://eddiecarbin.com/raspberry-pi-custom-splash-screen-with-blank-screen-on-boot/
